src/qrcode/StyledQRGen.tsx

```diff
@@ -0,0 +1,19 @@
+import QRCode from 'react-native-qrcode-svg'
+import React from 'react'
+import { SVG } from 'src/send/actions'
+
+function StyledQRCode({
+  value,
+  size = 100,
+  svgRef,
+}: {
+  value: string
+  size: number
+  color?: string
+  backgroundColor?: string
```

Comment:
Would these params be used eventually? if so, should there be a todo and/or is this tracked in a separate ticket?

Reply:
I've removed these params for the time being, they were left over from my testing. I'll spin up a ticket to figure out proper styling for the new QR code.